climada/hazard/centroids/test/test_vec_ras.py

```diff
@@ -139,19 +139,18 @@
     14.375, 11.875, 12.29166667, 12.70833333, 13.125, 13.54166667, 13.95833333, 14.375
 ])
 
+def data_vector():
+    vec_data = gpd.GeoDataFrame(crs={'init': 'epsg:32632'})
+    vec_data['geometry'] = list(zip(VEC_LON, VEC_LAT))
+    vec_data['geometry'] = vec_data['geometry'].apply(Point)
+    vec_data['lon'] = VEC_LON
+    vec_data['lat'] = VEC_LAT
+    vec_data['value'] = np.arange(VEC_LAT.size) + 100
+    return vec_data.lat.values, vec_data.lon.values, vec_data.geometry
```

Comment:
Since this function is used in several classes in this module, it doesn't make sense to have it as a static method of one of the classes.